projects/storefrontlib/src/lib/cms-lib/address-book/address-card/address-card.component.spec.ts

```diff
@@ -31,7 +31,9 @@ describe('AddressCardComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [AddressBookModule, StoreModule.forRoot({})]
+      declarations: [AddressCardComponent],
+      imports: [StoreModule.forRoot({})],
```

Comment:
We removed all "StoreModule" from storefrontlib. So, please remove this import.

Reply:
Good catch.